seismic-app/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# base image
+FROM node:12.2.0-alpine
```

Comment:
@tudoramariei do we need version pinning to be so specific? Or we can use use something like `node:12-apline`. This should gives the stable v12 api + minor changes / improvements (with BC) + security fixes.